crates/ludtwig/src/check/rules/twig_logic_same_as.rs

```diff
@@ -0,0 +1,76 @@
+use ludtwig_parser::syntax::typed::{AstNode, TwigBinaryExpression};
+use ludtwig_parser::syntax::untyped::{SyntaxKind, SyntaxNode};
+
+use crate::check::rule::{CheckResult, Rule, RuleExt, RuleRunContext, Severity};
+
+pub struct RuleTwigLogicSameAs;
+
+impl Rule for RuleTwigLogicSameAs {
+    fn name(&self) -> &'static str {
+        "twig-logic-same-as"
+    }
+
+    fn check_node(&self, node: SyntaxNode, _ctx: &RuleRunContext) -> Option<Vec<CheckResult>> {
+        let binary = TwigBinaryExpression::cast(node.clone())?;
+        let binary_expr_op = binary.operator()?;
+
+        if binary_expr_op.kind() == SyntaxKind::TK_TRIPLE_EQUAL
```

Comment:
Just a minor thing: you don't have to spell out token names like that, I have written a macro so you can write them like `T!["==="]` 🙂